### PR TITLE
WF-688 Auth error in container

### PR DESCRIPTION
**Pull request by yenobi**
- [x] pr is not finished yet

### CODE REVIEW

**Comment by yenobi** [comment]
The registry was changed temporarily due to the missing of one of the widgets, used in the app. Need to check, how to fix it

**Comment by yenobi** [comment]
What is left for the moment of creating PR:
the redirect URI has the wrong value and identity validation did not allow such a redirect.

The value is `http://localhost:4200${PATHNAME}transactions`, but should be something like `http://localhost:4200$/${BASE_HREF}/select-context` (might be not exactly the same).

**Comment by kirillsud** [comment]
- [ ]  Merge after a new web-base version 0.0.17 will be released. This version contains support for API proxy

**Comment by santam85** [comment]
What is the error we are trying to fix here? We really cannot rely on proxying the API in web-base, as that would not be possible on a CDN, so I want to make sure there aren't other alternatives

**Comment by kirillsud** [comment]
> What is the error we are trying to fix here? We really cannot rely on proxying the API in web-base, as that would not be possible on a CDN, so I want to make sure there aren't other alternatives

@santam85 I want to test `GSA` inside a Docker container based on `web-base` image

File: apps/golden-sample-app/src/app/user-context/user-context.component.ts [hunk 1]
```diff
@@ -1,6 +1,13 @@
 import { Component } from '@angular/core';
 import { LOGOUT } from '@backbase/foundation-ang/web-sdk';
 import { OAuthService } from 'angular-oauth2-oidc';
+import { UserContextGuard } from './user-context.guard';
+
+/**
+ * The select-context-widget requires a global reloadPage function that returns
+ * a Promise before executing an "spa:" navigation on context selection success.
+ */
+(window as any).reloadPage = () => Promise.resolve();
```
[the remark this answers is not in the document]
Comment (kirillsud) [reply apps/golden-sample-app/src/app/user-context/user-context.component.ts]: Done

**Review by santam85 — CHANGES_REQUESTED** [review]
I don't want to exemplify reverse proxing of APIs in the golden sample. Shall I reject this one?

**Pull request closed without merging**